### ISSUE TITLE
Action Text is missing after the repo was archived

### ISSUE THREAD

**Issue by IamNaN** [rails/rails#34881]
### Steps to reproduce
1. Install rails edge.
2. Try to add Action Text following [Edge Guides](https://edgeguides.rubyonrails.org/action_text_overview.html)

### Expected behavior
I can complete the installation instructions, Trix is added and Action Text works.

### Actual behavior
`rails -T` does not list the `action_text:install` task.
My app complains method missing on `has_rich_text` for the models that have it. (I was using Action Text before the repo was archived.)

### System configuration
**Rails version**: 6.0.0.alpha

**Ruby version**: 2.5.3p105

**More info:** I have been working with Rails 6 for a couple of months. `bundle update` failed for the first time when the rails/action_text repo was recently archived. So it seems like Action Text has not been fully merged into Rails yet. Thinking that maybe my Gemset was stale, I tried `bundle pristine`. (Trix is still installed and updates.)

Given that the repo was archived, I can't help with a PR or even give more help than I have.

**Comment by georgeclaghorn** [comment]
Do you selectively load Rails frameworks in `config/boot.rb`?

**Comment by IamNaN** [comment]
Hi George,

tldr; Got it, thanks!

Congrats and thanks for becoming a member of the core team.

Okay, in `application.rb` I had restricted the required frameworks, so Action Text wasn't included.
```require "rails/all"
# Pick the frameworks you want:
require "active_model/railtie"
require "active_job/railtie"
require "active_record/railtie"
require "active_storage/engine"
require "action_controller/railtie"
require "action_mailer/railtie"
require "action_view/railtie"
```

**Comment by georgeclaghorn** [comment]
> It looks like there isn't a railtie for action_text, so what do I include?

Action Text comes with an engine instead of a railtie, like Active Storage and Action Mailbox:

```ruby
require "action_text/engine"
```
